.github/workflows/ci.yaml

```diff
@@ -6,7 +6,7 @@ on:
   pull_request: {}
   push:
     branches:
-    - main
+      - main
```

Comment:
what's up with all these added spaces in the diff?

Reply:
My B. I didn't notice these, must be VS code triggering some yaml auto-formatting. I'll fix it up.